config/firebase.ts

```diff
@@ -9,6 +8,8 @@ import { getAnalytics } from "firebase/analytics"
 const firebaseConfig = {
   apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
   authDomain: "truecrime-6f10f.firebaseapp.com",
+  databaseURL:
+    "https://truecrime-6f10f-default-rtdb.europe-west1.firebasedatabase.app",
```

Comment:
Should this be in env?

Reply:
After your comment, I looked at other repos and decided that it would be best to use env variables to store all the configuration details not just the apiKey.  I earlier on unfortunately already committed the .env file before adding it to .gitignore so that's why it's currently visible.